### ISSUE TITLE
monitoring_uptime_check_config: Add validate_ssl to http_check

### ISSUE THREAD

**Issue by dustinblackman** [terraform-providers/terraform-provider-google#4631]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment. If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.

### Description

Currently Stackdriver's uptime check configs don't enable validating SSL certificates by default. The parameter is available as `validateSsl` within `httpCheck`.

https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.uptimeCheckConfigs#httpcheck

![Screen Shot 2019-10-08 at 11 21 18 AM](https://user-images.githubusercontent.com/5246169/66409239-4c058900-e9be-11e9-8f3d-0bd36dc6ce33.png)

### New or Affected Resource(s)

* google_monitoring_uptime_check_config

### Potential Terraform Configuration

```tf
resource "google_monitoring_uptime_check_config" "default" {
  display_name = "my-name"
  project      = "my-project"
  timeout      = "30s"
  period       = "60s"

  http_check {
    path    = "/"
    port    = "443"
    use_ssl = true
    validate_ssl = true
  }

  monitored_resource {
    type = "uptime_url"

    labels = {
      project_id = "project-id"
      host       = "google.com"
    }
  }
}
```

### References

https://github.com/GoogleCloudPlatform/magic-modules/pull/2441

**Comment by stepanstipl** [comment]
There is also already an auto-generated PR opened against this provider - https://github.com/terraform-providers/terraform-provider-google/pull/4637 (and beta - https://github.com/terraform-providers/terraform-provider-google-beta/pull/1243)

**Comment by dustinblackman** [comment]
@stepanstipl Yep, from the PR made on magic modules. https://github.com/GoogleCloudPlatform/magic-modules/pull/2441

**Comment by megan07** [comment]
Thanks @dustinblackman for opening this issue! It's been closed with https://github.com/GoogleCloudPlatform/magic-modules/pull/2441 and was released in version 2.20.0.
